### ISSUE TITLE
Global Replay Upload Failure - Record Mismatch

### ISSUE THREAD

**Issue by sneak-it** [KZGlobalTeam/gokz#384]
Seem to be getting this error log frequently ever since the 3.4.0 update.

```
L 10/12/2022 - 10:55:11: [gokz-global.smx] Failed to upload replay to the global API. Record mismatch.
L 10/12/2022 - 10:55:12: [SM] Exception reported: Invalid data pack handle 0 (error 4).
L 10/12/2022 - 10:55:12: [SM] Blaming: gokz-global.smx
L 10/12/2022 - 10:55:12: [SM] Call stack trace:
L 10/12/2022 - 10:55:12: [SM]   [0] DataPack.WriteString
L 10/12/2022 - 10:55:12: [SM]   [1] Line 125, gokz-global\send_run.sp::SendReplay
L 10/12/2022 - 10:55:12: [SM]   [2] Line 89, gokz-global\send_run.sp::OnReplaySaved_SendReplay
L 10/12/2022 - 10:55:12: [SM]   [3] Line 300, N:\sm 1.11\gokz-global.sp::GOKZ_RP_OnReplaySaved
L 10/12/2022 - 10:55:12: [SM]   [5] Call_Finish
L 10/12/2022 - 10:55:12: [SM]   [6] Line 52, gokz-replays\api.sp::Call_OnReplaySaved
L 10/12/2022 - 10:55:12: [SM]   [7] Line 523, gokz-replays\recording.sp::SaveRecordingOfRun
L 10/12/2022 - 10:55:12: [SM]   [8] Line 247, gokz-replays\recording.sp::Timer_EndRecording
L 10/12/2022 - 10:59:45: Error log file session closed.
```

**Comment by Sikarii** [comment]
#383 fixes this I believe.

**Comment by sneak-it** [comment]
Still getting several record mismatch errors, not sure if this is known or not. The exception seems to have been fixed with #333

**Comment by zealain** [comment]
Fixed based on the comments
